src/content/docs/contributing/modules.md

```diff
@@ -1082,12 +1082,35 @@ A PR review is the process of examining a new modules' submission or the changes
 - Ensure that temporary unzipped files are removed to avoid mitigating benefits and worsening problems.
 - Ensure that large outputs are compressed with the correct tool (follow guidelines for gzip vs bzip2 vs other options).
 
-#### In `../tests/modules/nf-core/modulename/main.nf` and `../tests/modules/nf-core/modulename/meta.yml`:
+#### In `modules/nf-core/modulename/meta.yml`:
 
-- Check that there are tests for all outputs, including optional ones.
-- Check that the `meta.yml` file has correct documentation links and patterns of files.
+- Check that the file has correct documentation links and patterns of files.
 - Run the tool help and check that important input (usually optional) has not been missed.
```

Comment:
that's quite an overkill imo... are people really doing this? I like more of an organic growth of inputs tbh

Reply:
That was already there :grin: do you want me to remove it?

Reply:
yes, please.